Source/APIClient/Request.swift

```diff
@@ -18,32 +18,18 @@ public enum HTTPMethod: String {
 public protocol Request {
     static var baseURL: String { get }
     static var acceptHeader: String? { get }
-    static var authToken: String? { get set }
+    static var authToken: String? { get }
```

Comment:
That's totally fine, users can make the `authToken` property settable if they want. I could also see an implementation that always pulls the token from some other object, like a keychain or something, and in that case there wouldn't necessarily be a setter. Or like my `TestRequest` mock object, where `authToken` is just a computed property that returns a constant value all the time.